src/www/index.html

```diff
@@ -148,149 +148,180 @@
       color: green;
     }
 
-    @media(max-width: 400px){
+    @media (max-width: 400px) {
       .custom-menu {
-          width: 250px;
+        width: 250px;
       }
     }
   </style>
 </head>
 
 <body>
-  <div class="custom-navbar">
-    <a href='.' class="custom-navtitle">
-      <strong>Open Event Webapp Generator</strong>
-    </a>
-    <div class="custom-menubutton">
-      <i class="glyphicon glyphicon-th"></i>
-    </div>
+<div class="custom-navbar">
```

Comment:
Please do not change the indentation, this is incorrect indentation.

Reply:
Oh,I was testing some changes and finally ran code reformatter with 2 indent spaces.Can you please once have a look ?